relayer-cli/src/commands/create/channel.rs

```diff
@@ -46,53 +46,61 @@ static HINT: &str = "Consider using the default invocation\n\nhermes create chan
 #[clap(disable_version_flag = true)]
 pub struct CreateChannelCommand {
     #[clap(
+        long = "a-chain",
         required = true,
+        value_name = "A_CHAIN_ID",
         help = "Identifier of the side `a` chain for the new channel"
     )]
     chain_a: ChainId,
 
     #[clap(
-        short,
-        long,
+        long = "b-chain",
+        value_name = "B_CHAIN_ID",
         help = "Identifier of the side `b` chain for the new channel"
     )]
     chain_b: Option<ChainId>,
 
-    /// Identifier of the connection on chain `a` to use in creating the new channel.
+    #[clap(
+        long = "a-conn",
+        value_name = "A_CONNECTION_ID",
+        help = "Identifier of the connection on chain `a` to use in creating the new channel."
+    )]
     connection_a: Option<ConnectionId>,
 
     #[clap(
-        long,
+        long = "a-port",
         required = true,
+        value_name = "A_PORT_ID",
         help = "Identifier of the side `a` port for the new channel"
     )]
     port_a: PortId,
 
     #[clap(
-        long,
+        long = "b-port",
         required = true,
+        value_name = "B_PORT_ID",
         help = "Identifier of the side `b` port for the new channel"
     )]
     port_b: PortId,
 
     #[clap(
-        short,
-        long,
+        long = "order",
+        value_name = "ORDER",
         help = "The channel ordering, valid options 'unordered' (default) and 'ordered'",
         default_value_t
     )]
     order: Order,
 
     #[clap(
-        short,
-        long = "channel-version",
+        long = "chan-version",
         alias = "version",
```

Comment:
Might as well remove this alias now because it's ambiguous with clap's built-in `--version` flag.